.gitlab-ci.yml

```
@@ -273,6 +273,9 @@ variables:
 .if_deploy_on_rc_tag_on_beta_repo_branch: &if_rc_tag_on_beta_repo_branch
   if: $DEPLOY_AGENT == "true" && $BUCKET_BRANCH == "beta" && $CI_COMMIT_TAG =~ /^[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+$/
 
+.if_scheduled_main: &if_scheduled_main
+  if: $CI_PIPELINE_SOURCE != "schedule" && $CI_COMMIT_BRANCH == "main"
```

I think this is what you meant?

```suggestion
  if: $CI_PIPELINE_SOURCE == "schedule" && $CI_COMMIT_BRANCH == "main"
```

---

Yeah, I tried the inverse originally and that didn't do what I wanted. Thanks!